src/redux/actions/__tests__/orgUnitGroupSets.test.js

```diff
@@ -0,0 +1,89 @@
+import { mockStore } from '../../../utils/test-helpers/mockStore'
+import {
+    actionTypes,
+    loadGroupSetOptions,
+    loadingGroupSetsErrorDefaultMessage,
+} from '../orgUnitGroupSets'
+import { getOrgUnitGroupSets } from '../../../utils/api'
+import * as feedbackTypes from '../../../utils/feedbackTypes'
+import { ACTION_TYPES as feedbackActionTypes } from '../feedback'
+
+jest.mock('../../../utils/api', () => ({
+    getOrgUnitGroupSets: jest.fn(),
+}))
+
+describe('Actions - organisationUnits GroupSets', () => {
+    let store
+    beforeEach(() => {
+        store = mockStore({})
+    })
+
+    const mockShowErrorFeedback = errorMessage => ({
+        type: feedbackActionTypes.FEEDBACK_SHOW_SNACKBAR,
+        payload: { message: errorMessage, type: feedbackTypes.ERROR },
+    })
+
+    const mockLoadingGroupSetsStart = () => ({
+        type: actionTypes.LOADING_GROUP_SETS_START,
+    })
+
+    const mockLoadingGroupSetsSuccess = groupSets => ({
+        type: actionTypes.LOADING_GROUP_SETS_SUCCESS,
+        payload: groupSets,
+    })
+
+    const mockLoadingGroupSetsError = errorMessage => ({
+        type: actionTypes.LOADING_GROUP_SETS_ERROR,
+        payload: errorMessage,
+    })
```

Comment:
In most action tests, you are now directly using with the actual exported actionCreators, which is good. However, here you are still using the mocked functions. Any particular reason for this?